build/test/TestPackageProjects.targets

```diff
@@ -109,11 +109,13 @@
                           Condition=" '$(IsDesktopAvailable)' == 'True' ">
         <Name>dotnet-desktop-binding-redirects</Name>
         <IsTool>True</IsTool>
+        <IsMsbuild>True</IsMsbuild>
         <IsApplicable>$(DesktopAvailable)</IsApplicable>
         <VersionPrefix>1.0.0-rc-</VersionPrefix>
         <VersionSuffix>$(TestPackageBuildVersionSuffix)</VersionSuffix>
         <Clean>True</Clean>
         <Frameworks>net451</Frameworks>
+        <Runtime>win7-x86</Runtime>
```

Comment:
I don't think you can do this? How would it work on other platforms? Should you use this $(CoreCLRRid) instead?